Sources/TensorFlow/Optimizers/MomentumBased.swift

```diff
@@ -33,8 +33,7 @@ import Numerics
 public class RMSProp<Model: Differentiable>: Optimizer
 where
   Model.TangentVector: VectorProtocol & PointwiseMultiplicative
-    & ElementaryFunctions & KeyPathIterable,
-  Model.TangentVector.VectorSpaceScalar == Float
+    & ElementaryFunctions & KeyPathIterable
```

Comment:
Nifty Vim search-and-replace command: `:%s/,\_s*Model.TangentVector.VectorSpaceScalar == Float//g`.

Apparently the `\_s` pattern matches a whitespace, tab, or newline character [(source)](https://vim.fandom.com/wiki/Search_across_multiple_lines).